jsdoc.conf.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Let's also rename this file for consistency with `newman`: `.jsdoc-config.json`

Ref: https://github.com/postmanlabs/newman/blob/develop/.jsdoc-config.json